sdk/vaa/quorum_test.go

```diff
@@ -55,3 +55,48 @@ func TestCalculateQuorum(t *testing.T) {
 		})
 	}
 }
+
+func FuzzCalculateQuorum(f *testing.F) {
+	// Add examples to our fuzz corpus
+	f.Add(-25)
+	f.Add(0)
+	f.Add(1)
+	f.Add(2)
+	f.Add(4)
+	f.Add(8)
+	f.Add(16)
+	f.Add(32)
+	f.Add(64)
+	f.Add(128)
+	f.Fuzz(func(t *testing.T, numGuardians int) {
+		// These are known cases, which the implementation will panic on and/or we have explicit
+		// unit-test coverage of above, so we can safely ignore it in our fuzz testing
+		if numGuardians <= 0 {
+			t.Skip()
+		}
+
+		// Let's determine how many guardians are needed for quorum
+		num := CalculateQuorum(numGuardians)
+
+		// Let's always be sure that there are enough guardians to maintain quorum
+		if num > numGuardians {
+			t.Errorf("%v", "fuzz violation: quorum cannot be acheived because we require more guardians than we have")
+		}
```

Comment:
Consider using [assert.Less()](https://pkg.go.dev/github.com/stretchr/testify/assert#Less)
```suggestion
		assert.Less(t, num, numGuardians, "fuzz violation: quorum cannot be achieved because we require more guardians than we have")
```